api/net/Areas/Services/Controllers/ReportController.cs

```diff
@@ -99,6 +98,27 @@ public async Task<IActionResult> FindContentForReportIdAsync(int id, int? reques
         return new JsonResult(results);
     }
 
+    /// <summary>
+    /// Get the current instance for the specified report 'id'.
+    /// </summary>
+    /// <param name="id"></param>
+    /// <param name="requestorId"></param>
+    /// <returns></returns>
+    [HttpGet("{id}/instance")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ReportInstanceModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType((int)HttpStatusCode.NoContent)]
+    [SwaggerOperation(Tags = new[] { "Report" })]
+    public IActionResult GetCurrentInstance(int id, int? requestorId)
```

Comment:
Needed to add this endpoint so that the Reporting Service has a way to get the linked report content.